lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -1,86 +1,85 @@
 """
 axis_artist.py module provides axis-related artists. They are
 
- * axis line
- * tick lines
- * tick labels
- * axis label
- * grid lines
-
-The main artist class is a AxisArtist and a GridlinesCollection. The
-GridlinesCollection is responsible for drawing grid lines and the
-AxisArtist is responsible for all other artists. The AxisArtist class
+* axis line
+* tick lines
+* tick labels
+* axis label
+* grid lines
+
+The main artist class are `AxisArtist` and a `GridlinesCollection`. The
```

Comment:
~~~
The main artist classes are `AxisArtist` and `GridlinesCollection`. While
`GridlinesCollection` is responsible for drawing grid lines, `AxisArtist`
is responsible for all other artists. `AxisArtist` has attributes that are 
associated with each type of artists.
~~~